go.mod

```diff
@@ -2,8 +2,12 @@ module github.com/rancher/rancher
 
 go 1.20
 
-// on release remove this wrangler replace and use the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+replace (
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
```

Comment:
Remove and update commit when https://github.com/rancher/aks-operator/pull/375 is merged.

Reply:
It was merged, please rebase